rocm.spec

```diff
@@ -56,6 +56,14 @@ Source29: https://%{repository}/%{repoversion}/main/rccl-2.22.3.60403-128.el%{rh
 Source30: https://%{repository}/%{repoversion}/main/rccl-devel-2.22.3.60403-128.el%{rhel}.%{_arch}.rpm
 Source31: https://%{repository}/%{repoversion}/main/rocshmem-devel-2.0.1.60403-128.el%{rhel}.%{_arch}.rpm
 
+# FIXME: Remove following source/fix once it has been integrated in to ROCM
+# [CUDA][HIP] Add a __device__ version of std::__glibcxx_assert_fail()
+%ifarch x86_64
+%if 0%{?rhel} < 10
```

Comment:
@smuzaffar why not for RHEL 10 ?

Reply:
ah my bad, as ROOT636/el10 IBs were not failing so I thought that we do not need this fix for el10 but I now realize that ROOT636/el10 did not have this issue as there was no rocm distribution for el10. 
Yes this should be applied to el10 too if rocm has not included the latest llvm in its distribution